docs/oauth2.md

```diff
@@ -62,149 +71,79 @@ auth_backends.1 = rabbit_auth_backend_oauth2
 
 Next, let's take a look at the workflows the OAuth 2 plugin supports.
 
-### Authorization Workflow {#authorization-workflow}
-
-This plugin does not communicate with any OAuth 2.0 provider. It decodes an access token provided by the client and authorises a user based on the data stored in the token.
-
-The token can be any [JWT token](https://jwt.io/introduction/) which contains the `scope` and `aud` fields. The way the token was issued (such as what grant type was used) is outside of the scope of this plugin.
-
 ### Prerequisites {#prerequisites}
 
 To use this plugin, all RabbitMQ nodes must be
 
 1. [configured to use the rabbit_auth_backend_oauth2 backend](./access-control).
-2. configured with a resource service ID (`resource_server_id`) that matches the scope prefix (e.g. `rabbitmq` in `rabbitmq.read:*/*`).
-3. configured with a signing key used by RabbitMQ to validate the JWT token signatures.
-
-JWT Tokens presented to RabbitMQ for authentication must
-
-1. be digitally signed with either a symmetric or asymmetric key.
-2. have a value in the `aud` field that matches `resource_server_id` value.
-
-
-### Authorization Flow {#authorization-flow-with-scopes}
-
-1. Client requests an `access_token` from the OAuth 2.0 provider,
-2. Token **scope** returned by OAuth 2.0 provider must include RabbitMQ resource scopes that follow a convention used by this plugin: `configure:%2F/foo` means "configure permissions for 'foo' in vhost '/'") (`scope` field can be changed using `extra_scopes_source` in **advanced.config** file.
-3. Client passes the token as password when connecting to a RabbitMQ node. **The username field is ignored**.
-4. The translated permissions are stored as part of the authenticated connection state and used the same way permissions from RabbitMQ's internal database would be used.
-
-## Usage {#usage}
+2. configured with the resource service ID (`resource_server_id`). The RabbitMQ cluster becomes an OAuth 2.0 resource and this is its identifier.
```

Comment:
Will a user know how to do this if not already done - do we need to link to docs?

Reply:
There is commit where i added a link to the section that covers it.